src/grblogtools/grblogtools.py

```diff
@@ -450,6 +450,41 @@ def get_log_info(loglines, verbose=False):
             if result:
                 values["Cuts: " + result["Name"].strip()] = int(result["Count"])
 
+    # NoRel log
+    norel_log_start = re.compile("Starting NoRel heuristic")
+    norel_first_line, result = _regex_first_match(
+        loglines, [norel_log_start], reverse=False
+    )
+    if norel_first_line and norel_first_line < len(loglines) - 1:
+        norel_last_line = _get_last_nonempty_line(loglines, norel_first_line + 1)
+        norel_primal_regex = re.compile(
+            "Found heuristic solution:\sobjective\s(?P<Incumbent>[^\s]+)"
+        )
+        norel_elapsed_time = re.compile(
+            "Elapsed time for NoRel heuristic:\s(?P<NoRelTime>\d+)s"
```

Comment:
Why not just `Time`?

Reply:
```suggestion
            "Elapsed time for NoRel heuristic:\s(?P<Time>\d+)s"
```

Reply:
Sure - we can do Time.  This is just the norel clock though, it starts from zero when NoRel starts (I am pretty sure) not when the solve starts.  So we should just make this clear somewhere, maybe in the readme.